simpleLooping/binaryListToString.py

```diff
@@ -15,3 +15,15 @@
 # Ex:
 #   binaryListToString([1, 0, 1, 1])       # return "1101"
 #   binaryListToString([0, 1, 1, 0, 0, 1]) # return "100110"
+from typing import List
+
+def binaryListToString(lst: List[int]) -> str:
+    for num in lst:
+        if num != 0 and num != 1:
+            return -1
+
+    binaryStr = ""
+    for i in range(len(lst)):
+        binaryStr += '0' if lst[i] == 0 else '1'
```

Comment:
I have a feeling that for input [0, 1, 1, 0, 0, 1] this code will produce “011001”. I think you should rerun your unit test and make sure if it is the case. Because we want “100110”.

If it is in fact not work like I say, then you might do some change.

From “binaryStr += …” 

To “binaryStr = … + binaryStr”